orthoinference/runOrthoinference.sh

```diff
@@ -7,15 +7,17 @@ git pull
 ## Create new jar file with orthoinference code
 mvn clean compile assembly:single
 
+## set args - all required
+configPath="src/main/resources/config.properties"	# Path to main configuration file
+refSpecies="orig"	# Reference species (4-digit abbv)
+projSpecies=(prj1 prj2 prj3)	# Projected species (4-digit abbv); may contain multiple species, space-delimited
```

Comment:
Apologies for the nit-pick, but could the comments say "4-character" rather than "4-digit" just so it's clear the abbreviations are not numeric?

Reply:
Absolutely. I can change.